data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
+	blocked "The building supervisor at <destination> has [requisitioned, hired] more workers than you can carry. You will need <capacity> to transport these workers."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Vara Rakak"
+	passengers 75
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "The construction workers board your ship chattering excitedly about their chance to pass through the Eye to new worlds."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort with the construction workers hasn't landed. You realize you need to depart and wait for them to arrive."
+
+	on abort
+		event "wanderers: spera anatrusk colony" 40
+
+	on fail
+		event "wanderers: spera anatrusk colony" 40
+
+	on complete
+		dialog "The construction supervisor greets the <ship> as you land at the spaceport site. He hands you a credit chip for <payment> and immediately begins assigning work to the teams."
+		payment 750000
+
+mission "Wanderers: Spera Anatrusk Outfitter 2b"
+	landing
+	name "Building materials to <planet>"
+	description "Bring <cargo> from <stopovers> to <destination> to build a new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [requisitioned, purchased] more building materials than you can carry. You will need <capacity> to transport it."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Ek'lak'lai"
+	cargo "prefabricated building materials" 200
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+	
+	on stopover
+		dialog "The dockworkers are talking excitedly as they load the building materials. When you ask, the dock foreman tells you they are [thrilled, delighted] to be helping establish the first settlement in the new worlds."
+
+	on visit
+		dialog "You've arrived at <planet> but the escort carrying the <cargo> hasn't landed. You realize you need to depart and wait for it."
+
+	on abort
+		event "wanderers: spera anatrusk colony" 40
+
+	on fail
+		event "wanderers: spera anatrusk colony" 40
+
+	on complete
+		dialog "A team of workers unload the <cargo> while the foreman checks the manifest before handing you a credit chip for <payment>."
+		payment 200000
+
+mission "Wanderers: Spera Anatrusk Outfitter 2c"
+	landing
+	name "Heavy machinery to <planet>"
+	description "Bring <cargo> from <stopovers> to <destination> to build a new outfitter. Payment is <payment>."
+	blocked "The building supervisor at <destination> has [ordered, requisitioned] more equipment than you can carry. You will need <capacity> to transport it."
+	source "Vara K'chrai"
+	destination "Spera Anatrusk"
+	stopover "Varu Tek'kai"
+	cargo "heavy machinery" 185
+
+	to offer
+		has "Wanderers: Spera Anatrusk Outfitter 1: offered"
+
+	on stopover
+		dialog "You notice that each crate of machinery has been signed by the dock team that loaded it."
```

Comment:
```suggestion
		dialog "As the crates of machinery are loaded into cargo, you notice that each has been signed by the dock team that loaded it."
```
The single sentence seems a bit too sparse, just a little bit more makes it flow better.